docs/detections/rules-ui-manage.asciidoc

```diff
@@ -21,6 +18,22 @@ On the Rules page, you can:
 * <<import-export-rules-ui>>
 * <<troubleshoot-signals>>
 
+[float]
+[[sort-filter-rules]]
+=== Sort and filter the rules list
+
+To sort the rules list, click the *Rule*, *Last updated*, or *Enabled* column header.
+
+TIP: To sort by any other column, switch on the *Technical preview* toggle above the table. This experimental table view allows advanced sorting capabilities. If you experience performance issues when working with the table, you can turn this setting off.
+
+To filter the rules list, enter a search term in the search bar and press **Return**:
+
+* Rule name — Enter a word or phrase from a rule's name.
+* Index pattern — Enter an index pattern (such as `filebeat-*`) to display all rules that use it. 
+* MITRE ATT&CK tactic or technique — Enter a MITRE ATT&CK tactic name (such as `Defense Evasion`) or technique number (such as `TA0005`) to display all associated rules.
+
+NOTE: Searches for index patterns and MITRE ATT&CK tactics and techniques must match exactly, are case sensitive, and do _not_ support wildcards. For example, to find rules using the `filebeat-*` index pattern, the search term `filebeat-*` is valid, but `filebeat` and `file*` are not because they don't exactly match the index pattern. Likewise, the MITRE ATT&CK tactic `Defense Evasion` is valid, but `Defense`, `defense evasion`, and `Defense *` are not.
```

Comment:
```suggestion
NOTE: Searches for index patterns and MITRE ATT&CK tactics and techniques must match exactly, are case sensitive, and do _not_ support wildcards. For example, to find rules using the `filebeat-*` index pattern, the search term `filebeat-*` is valid, but `filebeat` and `file*` are not because they don't exactly match the index pattern. Likewise, the MITRE ATT&CK tactic `Defense Evasion` is valid, but `Defense`, `defense evasion`, and `Defense*` are not.
```